entries/s/spotify.com.json

```
@@ -4,7 +4,8 @@
     "url": "https://www.spotify.com/",
     "contact": {
       "facebook": "spotify",
-      "twitter": "SpotifyCares"
+      "twitter": "SpotifyCares",
+      "form": "https://support.spotify.com/contact-spotify-support/"
```

Given that their page is behind a login screen and only provides a link that opens the mail application, I suggest replacing the form URL with the email.

```suggestion
      "email": "support@spotify.com"
```